src/networking.c

```diff
@@ -476,7 +476,7 @@ void addReplyProto(client *c, const char *s, size_t len) {
 }
 
 /* Low level function called by the addReplyError...() functions.
- * It emits the protocol for a Redis error, in the form:
+ * It emits the protocol for a server error, in the form:
```

Comment:
```suggestion
 * It emits the protocol for an error reply, in the form:
```

This can be a client error or a server error.

A server error usually means that something failed on the server side. For example, the HTTP error codes use 400 for client errors, like trying to access a file not found (404), while server errors are 500 series (e.g. 500 internal server error).